src/bpftrace.cpp

```diff
@@ -1190,8 +1190,9 @@ int BPFtrace::run(BpfBytecode bytecode)
     }
   }
 
-  if (bt_verbose)
-    std::cerr << "Running..." << std::endl;
```

Comment:
Any reason to remove the running msg completely? It can be nice to know tracing has started. You can add a BEGIN for that ofc, but for cli oneliners thats annoying